infra/lib/fhir-converter-service.ts

```diff
@@ -97,18 +97,17 @@ export function createFHIRConverterService(
     interval: Duration.seconds(10),
   });
 
-  // hookup autoscaling based on 90% thresholds
   const scaling = fargateService.service.autoScaleTaskCount({
-    minCapacity: isProd(props.config) ? 2 : 1,
+    minCapacity: isProd(props.config) ? 2 : 0.5,
```

Comment:
the .5 makes sense for the DB (since it uses ACUs), but AFAIK it doesn't make sense for containers (ie there cannot be .5 of a container running)

Reply:
💯 , good catch!